test/unit/spec/narrative_core/kbaseNarrativeWorkspace-spec.js

```diff
@@ -80,8 +81,10 @@ define([
         });
 
         it('buildAppCodeCell should fail without a spec', () => {
+            spyOn(window, 'alert');
             const widget = new KBaseNarrativeWorkspace($node);
             expect(widget.buildAppCodeCell()).toBeUndefined();
+            expect(window.alert).toHaveBeenCalled();
```

Comment:
stop the annoying alert from being called (has to be manually dismissed if you're running the tests in the browser)